tests/test_generators/output/kitchen_sink_pydantic.py

```diff
@@ -2,21 +2,21 @@
 from datetime import datetime, date
 from enum import Enum
 from typing import List, Dict, Optional, Any
-from pydantic import BaseModel, Field
-from pydantic.dataclasses import dataclass
+from pydantic import BaseModel as PydanticBaseModel, Field
 
 metamodel_version = "None"
 version = "None"
 
-# Pydantic config and validators
-class PydanticConfig:
-    """ Pydantic config https://pydantic-docs.helpmanual.io/usage/model_config/ """
-
-    validate_assignment = True
-    validate_all = True
-    underscore_attrs_are_private = True
-    extra = 'forbid'
-    arbitrary_types_allowed = True  # TODO re-evaluate this
+# class IntermediateBaseModel(PydanticBaseModel):
+#    __slots__ = '__weakref__'
+    
+class BaseModel(PydanticBaseModel,
```

Comment:
Normally I would expect AdjectiveFoo is_a Foo, not Foo is_a AdjectiveFoo

Also: won't think cause issues if we have multiple yaml files - each one makes a fresh pydantic model each with a base class BaseModel. Formally these are separated and users can do from foo import BaseModel as FooBaseModel, but this is a bit awkward

Or maybe there is no need to import this and it's something that just exists as a convenience?

Reply:
an alternative is the pydantic analog of YAMLRoot in linkml-runtime, this could be an option, but I like the ability to make things purely standalone

Reply:
It's awkward because I seem to need these two shim layers. Renaming the import from BaseModel to PydanticBasemodel is just copied and pasted from the solution I found, but because we're generating all of the classes, it's not really necessary to preserve the name BaseModel. I could try bundling up the configuration kwargs to make that a little less ugly, and get down to just the IntermediateBaseModel, which could be maybe renamed to BaseModelShim?

I feel like the BaseModel probably isn't the only piece of inheritance that would break if generating a schema from multiple yaml files (we would need to do imports for any inheritance across yaml files, right?), maybe it would make more sense to have the generator take a collection of yaml files?